doc: document generate with working directory. (#41)
terramate-io/terramate-docs: cli/code-generation/index.md

## Generate code

The [`generate`](../reference/cmdline/generate.md) command generate files configured by the available code generation strategies.

```sh
terramate generate
```

> The command above generates files for all configurations found within the current directory and its subdirectories. To generate files only for configurations in specific subdirectories, use the `-C` (or `--chdir`) flag followed by the desired directory path. Alternatively, you can navigate to the subdirectory using `cd` and run the command there.

Reviewer: ```suggestion
The above command generates files for all configurations within the current directory and subdirectories. To generate files for configurations in a specific subdirectory, use the `-C` (or `--chdir`) flag followed by the desired directory path. Alternatively, navigate to the subdirectory using `cd` and run the command there.
```
